python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -0,0 +1,354 @@
+from __future__ import absolute_import, division, print_function
+
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import numpy
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.coadd.utils as coaddUtils
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.pipe.tasks.assembleCoadd import _subBBoxIter
+
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddConfig
+from lsst.pipe.tasks.assembleCoadd import CompareWarpAssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import CompareWarpAssembleCoaddConfig
+
+__all__ = ["DcrAssembleCoaddTask"]
+
+
+class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
+    dcrBufferSize = pexConfig.Field(
+        dtype=int,
+        doc="Number of pixels to grow the subregion bounding box by.",
+        default=5,
+    )
+    dcrNSubbands = pexConfig.Field(
+        dtype=int,
+        doc="Number of sub-bands to forward model chromatic effects to fit the supplied exposures.",
+        default=3,
+    )
+    maxNIter = pexConfig.Field(
+        dtype=int,
+        doc="Maximum number of iterations of forward modeling.",
+        default=8,
+    )
+    convergenceThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Maximum number of iterations of forward modeling.",
+        default=0.001,
+    )
+    assembleStaticSkyModel = pexConfig.ConfigurableField(
+        target=AssembleCoaddTask,
+        doc="Task to assemble an artifact-free, PSF-matched Coadd to serve as a"
+            " naive/first-iteration model of the static sky.",
+    )
+
+    def setDefaults(self):
```

Comment:
I assume this is here because you're expecting to set some `Assemble` defaults by the time its time to merge.